test/tdigest_test.rb

```diff
@@ -78,4 +78,32 @@ def test_that_it_has_a_version_number
       end
     end
   end
+
+  describe '#push' do
+    it 'does not fail when nearest centroid mean_cumn is nil' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.